Sources/Flightdeck/Flightdeck.swift

```diff
@@ -1,6 +1,416 @@
-public struct Flightdeck {
```

Comment:
General comment: I don't quite understand the importance of the day/month split in how you hold events. I'd rather see them structured in an Array, with convenience methods (in the form of computed properties) to filter them by criteria. That improves the simplicity and readability quite a bit I think. 

If you require that format for sending them off to the server, you could transform them in that format once you build up your request.

Reply:
Here's another take on above comment: 

Instead of using an EventsTrackedBefore struct that holds string events, why not have an Array that contains Event objects? 

Example:

`let events: [Event] = []`

You could then filter events by date like this:

`let todaysEvents = events.filter { $0.date >= Calendar.current.startOfDay(for: .now)  }`

Or even better: have a computed property that dynamically does that:

`var todaysEvents { events.filter { $0.date >= Calendar.current.startOfDay(for: .now)  }`